app/assets/javascripts/admin/payments/directives/stripe_elements.js.coffee

```diff
@@ -18,19 +20,21 @@ angular.module('admin.payments').directive "stripeElements", ($injector, AdminSt
             color: '#5c5c5c'
             '::placeholder':
               color: '#6c6c6c'
-      card.mount('#card-element')
+      card.mount(elem.find('.card-element').get(0))
```

Comment:
Not sure why the id was not working... do we have multiple '#card-element'? but then why always taking the first one `.get(0)`? And should we remove the css id and keep only the class to avoid warnings?

Sorry my 2 cents without all the context or a lot of background/knowledge of the code...

Reply:
The `id` was working, but the idea here is to use multiple stripe elements in the same page. Before the PR, the card was always mounted to the (I guess) last `#card-element`: we actually want to mount the card to the right element, ie the one which is handled by this angularjs directive (we search for `.card-element` inside the child of `elem`, ie. the element linked by the angularjs directive).

`get(0)` is the get the first DOM element since `elem.find('whetever here')` returns an array of jQuery object. There must be only one element matching `card-element` inside the `elem` DOM element.

Is that more clear?

Reply:
Yes ok inside the current directive there will be only one so ok for that.

But I think we should then get rid of the id because it's not html-y correct to have multiple in the same page, or at least generate unique ids (either random or with the index of the current stripe element?).

Reply:
Yes, why not removing this id afterall.
I should be careful about the CSS and the specs, but it should be straightforward.

Reply:
See 59eb91c ;)

Reply:
Thanks :) I think it's cleaner especially if the label is not used!